tests/ord-aggregator/tests/handler_test.go

```diff
@@ -223,6 +203,41 @@ func TestORDAggregator(t *testing.T) {
 		httpClient := conf.Client(ctx)
 		httpClient.Timeout = 20 * time.Second
 
+		// Get vendors
+		respBody := makeRequestWithHeaders(t, httpClient, testConfig.ORDServiceURL+"/vendors?$format=json", map[string][]string{tenantHeader: {testConfig.DefaultTestTenant}})
+		numberOfGlobalVendors := len(gjson.Get(respBody, "value").Array())
+		expectedNumberOfVendors += numberOfGlobalVendors
+
+		// Get products
+		respBody = makeRequestWithHeaders(t, httpClient, testConfig.ORDServiceURL+"/products?$format=json", map[string][]string{tenantHeader: {testConfig.DefaultTestTenant}})
+		numberOfGlobalProducts := len(gjson.Get(respBody, "value").Array())
+		expectedNumberOfProducts += numberOfGlobalProducts
```

Comment:
Add some logs for easier troubleshooting of failing tests.

How many global products / vendors are found and what is the new expected number.